docs/conf.py

```diff
@@ -29,13 +29,13 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
+import importlib.metadata as importlib_metadata
```

Comment:
Why not just use `from towncrier import __verison__` or something similar ?

Reply:
The changes in `doc` were recycled but now they are no longer relevant. Reverted